README.md

```diff
@@ -1,46 +1,54 @@
-Set up and manage multiple organizations under a single GitHub Enterprise account using Infrastructure as Code (IaC) principles with Terraform. Using these tools you can establish consistent security best practices by default and leverage drift detection to identify unauthorized changes and ensure your configurations remain secure. 
-### Centralize Control:
-Gain a single pane of glass view for managing configurations, teams, and cloud provider settings across all your organizations for better reporting and decision making. 
-### Rapid Rollouts:
-Streamline updates and enforce security policies consistently across your entire enterprise, eliminating the need for time-consuming manual intervention by individual organizations. With a single update you can be confident that all your organizations are up to date at the same time and by automating repetitive tasks and streamlining updates your teams can focus on innovation rather than lengthy and repetitive updates. 
-### Reduce Security Risks:
-Minimize the attack surface by enforcing consistent security policies across all your organizations. 
-This project is Open Source and customizable. Released under the [MIT license](./LICENSE.md), you can tailor the toolkit to your specific needs and contribute to its ongoing development. 
-## Features:
+- **Centralize Control:** 
+  - Gain a comprehensive overview for managing organizations, repositories, and teams across your enterprise.
+
+- **Rapid Rollouts:** 
+  - Streamline updates and security policy enforcement across your entire enterprise, reducing the need for manual intervention. 
+  - Automate repetitive tasks, allowing teams to focus on innovation.
+
+- **Reduce Security Risks:** 
+  - Enforce consistent security policies across all organizations to minimize vulnerabilities and protect against attacks.
+
+This project is open-source and customizable, released under the [MIT license](./LICENSE.md). You're invited to adapt the toolkit to your specific needs and contribute to its development.
+
+## Detailed Features:
 
 ### Current:
-- Our solution leverages a GCP backend for infrastructure provisioning. (Future support for Azure and AWS coming soon!) 
-- Terraform and Terragrunt are the industry-standard tools used for managing infrastructure as code, including drift detection. 
-- Automate creation, deletion, and configuration of repositories (private and public). 
-- Efficiently manage teams and team memberships across organizations. 
-- Securely store and manage sensitive secrets within your GitHub environment. 
-- Enforce granular access control using Rulesets for branch protection. 
-### Future:
-- Multi-Cloud Support for seamless integration with Azure and AWS cloud providers. 
-- Enhanced Security features based on community needs. 
+- [x] **GCP Backend for Infrastructure Provisioning:** Current support for Google Cloud Platform, with future plans to integrate Azure and AWS for a multi-cloud approach.
```

Comment:
The only real value we have in the Upcoming section is the inclusion of multi-cloud, so I say either omit it here or let's just get rid of Upcoming altogether (it's basically just saying "we're working on stuff" and slows scrolling to the actual call to action of getting started).